src/ScottLogic.Internal.Training/ScottLogic.Internal.Training.Api/Controllers/TradesController.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using ScottLogic.Internal.Training.Matcher;
+
+namespace ScottLogic.Internal.Training.Api.Controllers
```

Comment:
Didn't see any tests for this. Only really needs one, but introduces mocking so worth having.